tests/integration/targets/dcnm_network/tests/dcnm/merged.yaml

```diff
@@ -370,6 +378,58 @@
   wait_for:
     timeout: 20
 
+- name: MERGED - setup - Clean up any existing network
+  cisco.dcnm.dcnm_network:
+    fabric: "{{ ansible_it_fabric }}"
+    state: deleted
+
+- name: MERGED - sleep for 20 seconds for DCNM to completely update the state
+  wait_for:
+    timeout: 20
+
+- name: MERGED - Create, Attach and Deploy Single Network with Multiple Switch Attach - Routing Tag Specified
+  cisco.dcnm.dcnm_network:
+    fabric: "{{ ansible_it_fabric }}"
+    state: merged
+    config:
+    - net_name: ansible-net13
+      vrf_name: Tenant-1
+      net_id: 7005
+      net_template: Default_Network_Universal
+      net_extension_template: Default_Network_Extension_Universal
+      vlan_id: 1500
+      routing_tag: 321
```

Comment:
We should also test that the default case when the `routing_tag` is not specified in the playbook results in the default value of `12345` being used.

Reply:
no even this is not possible, if we dont set the routing tag and the default value is set by dcnm can be seen only in dcnm manually.
Even the diffs from the api and the invocation doesnt have those details.

"invocation": {
        "module_args": {
            "config": [
                {
                    "attach": [
                        {
                            "deploy": true,
                            "ip_address": "10.122.84.110",
                            "ports": [
                                "Ethernet1/1",
                                "Ethernet1/2"
                            ]
                        }
                    ],
                    "deploy": false,
                    "gw_ip_subnet": "192.168.30.1/24",
                    "net_extension_template": "Default_Network_Extension_Universal",
                    "net_id": 7005,
                    "net_name": "ansible-net13",
                    "net_template": "Default_Network_Universal",
                    "vlan_id": 1500,
                    "vrf_name": "Tenant-1"
                }
            ],
            "fabric": "Fabric1",
            "state": "merged"
        }